### ISSUE TITLE
0.2.6 crashes on Mac (dmg build only)

### ISSUE THREAD

**Issue by bakman2** [thomasnordquist/MQTT-Explorer#117]
Yesterday Apple released an update, crashing perhaps related?
 
OS version:
MacOS 10.14.5 (18F132)
18.6.0 Darwin Kernel Version 18.6.0: Thu Apr 25 23:16:27 PDT 2019; root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64

**Comment by thomasnordquist** [comment]
Thanks for your help =)

Can you please be more specific?
- Does the App start?
- Does the App crashes on start?
  - if yes, what is the output when you execute `/Applications/MQTT\ Explorer.app/Contents/MacOS/MQTT\ Explorer` in your terminal.
- Installed from the App Store or from DMG?
- Have you seen specific scenarios when the App crashes?

**Comment by bakman2** [comment]
Installed from DMG found in releases here on GH. 

The app does not start, crashes on start:
```
./MQTT\ Explorer
[09:13:42.201] [info] App starting...
subscribing connection/add/mqtt
subscribing connection/remove
subscribing storage/store
subscribing storage/load
subscribing storage/clear
icon path /Applications/MQTT Explorer.app/Contents/Resources/app.asar/icon.png
[7335:0517/091342.478653:FATAL:gpu_data_manager_impl_private.cc(892)] The display compositor is frequently crashing. Goodbye.
[1]    7335 trace trap  ./MQTT\ Explorer
```

**Comment by bakman2** [comment]
Interestingly enough, the appstore version works.

**Comment by thomasnordquist** [comment]
It is hard to reproduce this issue. There has been a new electron release that might have a related fix.
https://github.com/electron/electron/releases/tag/v5.0.1
https://github.com/electron/electron/pull/18004

I'll definetly try to repruduce it and will update electron to 5.0.1 end of this month.

**Comment by thomasnordquist** [comment]
Should be fixed with electron >= 5.0.2
